scripts/install/install_macos.sh

```diff
@@ -631,14 +652,14 @@ uninstall()
   rm -f "/Library/LaunchDaemons/$SERVICE_NAME.plist" || error_exit "$LINENO" "Failed to remove service file /Library/LaunchDaemons/$SERVICE_NAME.plist"
   succeeded
 
-  info "Backing up config.yaml to config.bak.yaml"
-  cp "$INSTALL_DIR/config.yaml" "$INSTALL_DIR/config.bak.yaml" || error_exit "$LINENO" "Failed to backup config.yaml to config.bak.yaml"
+  info "Backing up effective.yaml to effective.bak.yaml"
+  cp "$INSTALL_DIR/effective.yaml" "$INSTALL_DIR/effective.bak.yaml" || error_exit "$LINENO" "Failed to backup effective.yaml to effective.bak.yaml"
   succeeded
```

Comment:
Let's change the path for effective.yaml here to be under the supervisor storage dir, that recently had changed.

Reply:
Actually, let's just remove this step. The effective.yaml is just given from BPOP, it's never something the user had configured right now. We can add it back in if we end up using it for standalone installs.